### PR TITLE
Tester

### PR DESCRIPTION
**Pull request by batshoes**

### CODE REVIEW

File: lib/test_invoke.rb [hunk 1]
```diff
@@ -0,0 +1 @@
+puts "hello world"
```
Comment (paniagua) [inline lib/test_invoke.rb:1]: this does nothing

File: lib/zendesk_call.rb [hunk 2]
```diff
@@ -8,7 +8,7 @@ def call(email)
 class GetTicketRequest
   def call(email, user_id)
     @user_id = user_id
-    @tickets = client.search(:query => "#{email}")
+    @tickets = client.search(:query => "#{email}", :reload => 'true')
```
Comment (paniagua) [inline lib/zendesk_call.rb:11]: why is the reload needed?

File: lib/test_invoke.rb (hunk 1 above)
Comment (batshoes) [inline lib/test_invoke.rb:1]: I meant to take that out. This was Justin's idea to see if the gem was being invoked.

File: lib/zendesk_call.rb (hunk 2 above)
Comment (batshoes) [inline lib/zendesk_call.rb:11]: I seemed to be getting cached results after taking it out. I had many engagements and was still receiving the same 3 tickets.

File: lib/test_invoke.rb (hunk 1 above)
Comment (paniagua) [inline lib/test_invoke.rb:1]: the idea was ok for debugging but it should not live in the code in git.

Comment (batshoes) [inline lib/test_invoke.rb:1]: Makes sense, I will remove it

File: lib/zendesk_call.rb (hunk 2 above)
Comment (paniagua) [inline lib/zendesk_call.rb:11]: does it have to be a `string` or it should be a `boolean` ?

Comment (batshoes) [inline lib/zendesk_call.rb:11]: Documentation had it listed as a string, which I thought was strange, but just mimicked it.

Comment (batshoes) [inline lib/zendesk_call.rb:11]: Actually, no it doesn't. I was wrong.
https://github.com/zendesk/zendesk_api_client_rb/wiki/Caching

Comment (paniagua) [inline lib/zendesk_call.rb:11]: :+1:

**Pull request merged**